tutorials/fundamentals/check-integer.md

```diff
@@ -0,0 +1,28 @@
+To check if a variable is an integer in JavaScript, use `Number.isInteger()`.
+`Number.isInteger()` returns `true` or `false` depending on the parameter provided.
+
+```javascript
+let example = 12.1;
+Number.isInteger(example); // false
+example = 12;
+Number.isInteger(example); // true
+example = Infinity;
+Number.isInteger(example); // false
+```
+The neat thing about `Number.isInteger()` is that you can pass mathmatical expressions which the function will evaluate and return the boolean result.
```

Comment:
Not necessary to elaborate on `Number.isInteger(4/2) === true`. Replace this sentence and example with an explanation of what happens with non-numeric values:

```javascript
Number.isInteger(null); // false
Number.isInteger('42'); // false
Number.isInteger(new Number(5)); // false
```